frontend/spec/selenium/spec/instances_and_containers_spec_obsolete.rb

```diff
@@ -1,419 +0,0 @@
-
-require_relative '../spec_helper'
-
-describe 'Resource instances and containers' do
-  before(:all) do
-    @repo = create(:repo, repo_code: "instances_test_#{Time.now.to_i}")
-    set_repo @repo
-
-    @resource = create(:resource)
-    @accession = create(:accession)
-    @accession_no_container = create(:accession)
-
-    @location_a = create(:location)
-    @location_b = create(:location,
-                         temporary: 'conservation')
-    container_location = build(:container_location,
-                               ref: @location_a.uri)
-    @container = create(:top_container,
-                        container_locations: [container_location])
-
-    # Some containers for searching
-    ('A'..'E').each do |l|
-      create(:top_container,
-             indicator: "Letter #{l}",
-             container_locations: [container_location])
-    end
-
-    run_all_indexers
-
-    @driver = Driver.get.login_to_repo($admin, @repo)
-  end
-
-  before(:each) do
-    @driver.navigate.to($frontend.to_s)
-  end
-
-  after(:all) do
-    @driver ? @driver.quit : next
-  end
-
-  it 'searches containers and performs bulk operations' do
-    @driver.navigate.to("#{$frontend}/top_containers")
-
-    @driver.find_element(id: 'q').send_keys('Letter')
-    @driver.find_element(css: 'input.btn').click
-
-    @driver.wait_for_ajax
-
-    # the search param is added to the download csv button
-    expect(@driver.find_element(class: 'searchExport').attribute('href')).to match(/q=Letter/)
-
-    results = @driver.find_element(id: 'bulk_operation_results')
-
-    expect(results.find_elements(css: 'tbody tr').length).to eq(5)
-
-    # Now sort by indicator
-    @driver.find_element(css: '#bulk_operation_results th:nth-child(5)').click
-
-    @driver.wait_for_ajax
-
-    expect(@driver.find_element(css: '#bulk_operation_results tbody tr:first-child td.top-container-indicator').text).to eq('Letter A')
-
-    @driver.find_element(css: '#bulk_operation_results tbody tr:first-child td:first-child input').click
-
-    # Now bulk update Letter A's ILD #
-    @driver.find_element(css: '.bulk-operation-toolbar:first-child a.dropdown-toggle').click
-    @driver.wait_for_dropdown
-
-    @driver.find_element(id: 'bulkActionUpdateIlsHolding').click
-
-    modal = @driver.find_element(id: 'bulkUpdateModal')
-
-    modal.find_element(id: 'ils_holding_id').send_keys('xyzpdq')
-
-    modal.find_element(css: "button[type='submit']").click
-
-    modal = @driver.find_element(id: 'bulkUpdateModal')
-
-    expect do
-      modal.find_element_with_text('//div[contains(@class, "alert-success")]', /Top .+ updated/)
-    end.not_to raise_error
-
-    modal.find_element(css: '.modal-footer button').click
-
-    @driver.click_and_wait_until_gone(css: '#bulk_operation_results tbody tr:first-child td:last-child a:first-child')
-
-    expect(@driver.find_element(css: '.form-group:nth-child(3) div.label-only').text).to eq('xyzpdq')
-
-  end
-
-  it "performs bulk indicator update" do
```

Comment:
I'll note that this test is ignored [in the file that replaced this file](https://github.com/archivesspace/archivesspace/blob/4f85a92f89625ce0eca39a4eb4380975b5de2689/frontend/spec/features/top_containers_spec.rb#L83-L93) with a comment about this test in this file is incomplete.

> # the original selenium test appears to have been left unfinished